src/react/ZenkoUI.js

```diff
@@ -21,19 +21,27 @@ function ZenkoUI() {
   const isConfigLoaded = useSelector(
     (state: AppState) => state.auth.isConfigLoaded,
   );
+
   const configFailure = useSelector(
     (state: AppState) => state.auth.configFailure,
   );
   const configFailureErrorMessage = useSelector((state: AppState) =>
     state.uiErrors.errorType === 'byComponent' ? state.uiErrors.errorMsg : '',
   );
   const theme = useSelector((state: AppState) => state.uiConfig.theme);
+  const [configLoaded, setConfigLoaded] = useState(isConfigLoaded);
 
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch(loadAppConfig());
-  }, [dispatch]);
+  }, []);
+
+  useEffect(() => {
```

Comment:
for me the `isConfigLoaded` is sufficient, since it come from a reactive context (redux) it should be updated, and force the render everytime it change without the need to use useEffect and useState.

Reply:
Good catch